050-Manage/030-migrations.mdx

```diff
@@ -73,10 +73,131 @@ Migration files should not be modified. If externally modified, the record will
 
 Migrations must be ordered. Ordering is guaranteed by the `parent` and the ledger file, which is stored in `.xata/migrations/.ledger`. The ledger file is an append only file that lists all migration files in the correct order.
 
-## UI Migration Editor for pgroll operations
+## Multi-version Schema Migrations (Postgres enabled only)
 
-Under the Schema menu, the Web UI provides a Migration Editor to create migrations using [pgroll](https://github.com/xataio/pgroll) operations.
+For Postgres enabled branches, the Web UI provides a Migration Editor to create [multi-version schema migrations](https://xata.io/blog/schema-multi-version-launch-week) using [pgroll](https://github.com/xataio/pgroll) operations.
 
-In the editor you can provide content for the pgroll operations array and start a migration. Once submitted, the migration will enter the Active state and must be manually completed with the "Complete migration" action button before further schema edits can take place. While in Active state, the migration can also be rolled back.
+Multi-version schema migrations address the pain point of having to keep your application code in sync with your database schema by allowing you two active database schemas at once. With multi-version schema migrations, running a migration means being able to preview the new version of your schema alongside the old one. Reading and writing data in both schemas continues to work normally. As a result of having two active schema versions, rollbacks become seamless operations which is particularly useful in the case of an unwanted schema change or backfill.
+
+<Alert status="warning">
+  Currently multi-version schema migrations are only supported via the Migration Editor in the UI. Migrations made via
+  the Table View and Schema View will complete automatically.
+</Alert>
+
+![The Migration Editor](images/migrations-multiversion/migration-editor.png)
+
+Some common schema operations are:
+
+<TabbedCode tabs={['create_table', 'add_column', 'rename_table', 'alter_column', 'drop_table', 'drop_column']}>
```

Comment:
I'm not sure where this would live, but it might be worth defining what the `up` and `down` parameters are explicitly doing.

Reply:
Ah i see it later on, but maybe it's worth explicitly stating that these triggers are for upstream / downstream changes, require SQL, etc.

Reply:
Added some info about the up and down triggers here